doc/metrics.md

```diff
@@ -95,6 +95,13 @@ Then add this certificate to the list of trusted clients, specifying the type as
 
     lxc config trust add metrics.crt --type=metrics
 
+If requiring TLS client authentication isn't possible in your environment, the `/1.0/metrics` API endpoint can be made available to unauthenticated clients. While not recommended, this might be acceptable if you have other controls in place to restrict who can reach that API endpoint. To disable the authentication on the metrics API:
```

Comment:
Can you add a line break before the word "While".

This won't result in any change of generated output in the docs itself, but makes review easier.

Reply:
@ru-fu would you say that we should add a line break after each full stop?